docs/README.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line MD041 -->
+[![Pillarbox logo](README-images/logo.jpg)](https://github.com/SRGSSR/pillarbox-documentation)
+
+This repository provides comprehensive documentation and specifications for the Pillarbox player solution, which
+supports Android, Apple, and Web platforms.
+
+Our goal is to ensure that developers and stakeholders have access to clear and detailed guidelines to facilitate the
+successful implementation and integration of Pillarbox across various projects.
+
+To access the documentation, please visit: [Pillarbox Documentation][pillarbox-docs].
+
+## How to Contribute
+
+We welcome contributions from everyone. The site is hosted on GitHub Pages and is powered by [docsify][docsify], served
+from the [public](../public) folder in this repository.
+
+Here's a brief overview of the project structure:
+
+- [public](../public): This directory serves as the homepage for the docsify site and contains all the documentation
+  files written in markdown.
+- [public/_sidebar.md](../public/_sidebar.md): This markdown file renders the sidebar of the page. Add links to your
+  documentation sections here.
+
+To create a new section or document, simply add a file under [public](../public) and include a link to it in
+the [_sidebar.md](../public/_sidebar.md) file.
+
+> [!TIP]
+> To test the site, simply open [public/index.html](../public/index.html) on your project.
+
+Please adhere to our [Contribution Guidelines](./CONTRIBUTING.md) when making a contribution.
+
+Feel free to [open issues][open-issues], [submit pull requests][submit-pr], or [share your ideas][discussions].
+
+## License
+
+For more information, please refer to the [LICENSE](../LICENSE) file.
```

Comment:
The license is missing but we can merge this PR and add it in another one.

Reply:
It's been hard work, but I've managed to add the license to this PR. That way we don't have to worry about it anymore.